src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -50,10 +50,11 @@ export type HostConfig<T, P, I, TI, C, CX> = {
 
   createInstance(type : T, props : P, rootContainerInstance : C, hostContext : CX, internalInstanceHandle : OpaqueNode) : I,
   appendInitialChild(parentInstance : I, child : I | TI) : void,
-  finalizeInitialChildren(parentInstance : I, type : T, props : P, rootContainerInstance : C) : void,
+  finalizeInitialChildren(parentInstance : I, type : T, props : P, rootContainerInstance : C) : boolean,
```

Comment:
True! What other sorts of work do you think `finalizeInitialChildren` might want to schedule? (I'd prefer to think of at least one other likely use case before adding a constant.)